sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/HttpConstants.java

```diff
@@ -272,6 +272,9 @@
     public static class StatusCodes {
         public static final int OK = 200;
         public static final int NOT_MODIFIED = 304;
+        // Success
+        public static final int MINIMUM_SUCCESS_STATUSCODE = 200;
+        public static final int MAXIMUM_SUCCESS_STATUSCODE = 299;
```

Comment:
What about 304 (NOT_MODIFIED) - does that not count in maximum success code?

Reply:
3xx are Redirection status codes.

2xx - success
3xx - redirection
4xx - client failure
5xx - service failure